subprojects/logging/src/main/java/org/gradle/internal/logging/console/BuildStatusRenderer.java

```diff
@@ -63,6 +66,9 @@ public void onOutput(OutputEvent event) {
             // if it has no parent ID, assign this operation as the root operation
             if (startEvent.getParentId() == null && BUILD_PROGRESS_CATEGORY.equals(startEvent.getCategory())) {
                 rootOperationId = startEvent.getOperationId();
+                if (buildStartTimestamp == 0) {
+                    buildStartTimestamp = startEvent.getTimestamp();
```

Comment:
What would look the oddest? Finishing with a time lower than the summary time or starting with a time higher than zero? We could propagate the `Clock` the same way we propagate the max workers and log level.